test/fixtures/rules/std_rule_cases/L028.yml

```diff
@@ -44,3 +44,24 @@ test_value_table_functions_do_not_require_qualification:
   configs:
     core:
       dialect: bigquery
+    rules:
+      L028:
+        force_enable: True
+
+test_object_references_1:
```

Comment:
Similar to L026 test comment, should we consider renaming these cases as:
* test_object_references_1_a
* test_object_references_1_b
* test_object_references_1_c